cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-acm-metrics-2ti-object-storage/externalsecret.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: open-cluster-management-observability-acm-metrics-2ti-object-storage
+  namespace: open-cluster-management-observability
+spec:
+  refreshInterval: 15s
+  secretStoreRef:
+    name: nerc-cluster-secrets
+    kind: ClusterSecretStore
+  target:
+    name: acm-metrics-2ti-object-storage
+    template:
+      type: Opaque
+      engineVersion: v2
+      data:
+        acm-metrics.yaml: |
+          type: s3
+          config:
+            bucket: "{{ .bucket }}"
+            endpoint: "{{ .endpoint }}"
+            insecure: false
+            access_key: "{{ .access_key }}"
+            secret_key: "{{ .secret_key }}"
+            http_config:
+              tls_config:
+                ca_file: "/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+  data:
+  - secretKey: bucket
+    remoteRef:
+      key: $ENV/$CLUSTER/open-cluster-management-observability/acm-metrics-2ti/object-storage    # the path-details will be replaced in the overlay patch
+      property: bucket
+  - secretKey: endpoint
+    remoteRef:
+      key: $ENV/$CLUSTER/open-cluster-management-observability/acm-metrics-2ti/object-storage
+      property: endpoint
+  - secretKey: access_key
+    remoteRef:
+      key: $ENV/$CLUSTER/open-cluster-management-observability/acm-metrics-2ti/object-storage
+      property: access_key
+  - secretKey: secret_key
+    remoteRef:
+      key: $ENV/$CLUSTER/open-cluster-management-observability/acm-metrics-2ti/object-storage
+      property: secret_key
```

Comment:
Do we really need all of these entries? We're mapping the key in the vault secret to the same key in the kubernetes secret.

Reply:
when you mentioning it, i guess not.
I just remembered, the line 29 `# the path-details will be replaced in the overlay patch`
and followed the steps before.
In my opinion these entries can be whatever the want to be, being replaced ion the overlay.

Or are you even talking about another area where they are already set?
@larsks